.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,47 @@
+name: deployement
+
+on:
+  workflow_run:
+    workflows: ["rust"]
+    branches: [master]
+    types: [completed]
+
+env:
+  DEPLOY_USER: at2
+  DEPLOY_HOST: srv1.c4dt.org
+
+jobs:
+  deploy-ns:
+    name: deploy AT2's node
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+
+      - uses: Swatinem/rust-cache@v1
+
+      - uses: actions-rs/cargo@v1
+        name: cargo build
+        with:
+          command: build
+          args: --bin server --features server
+
+      - uses: kielabokkie/ssh-key-and-known-hosts-action@v1.2.0
+        with:
+          ssh-host: ${{ env.DEPLOY_HOST }}
+          ssh-private-key: ${{ secrets.DEPLOY_SSH_KEY }}
+
+      - name: rsync repository
+        run: >
+          rsync --archive "$GITHUB_WORKSPACE/target/debug/server"
+          "${{ env.DEPLOY_USER }}@${{ env.DEPLOY_HOST }}:at2-node"
+
+      - name: rebuild docker-compose
+        run: >
+          ssh "${{ env.DEPLOY_USER }}@${{ env.DEPLOY_HOST }}"
```

Comment:
Does that also call `docker-compose down` first?

Reply:
Yes and no, it only `down` the newer images, why is usually the case in a deployement.

Reply:
Thanks - I was not aware of that.